precompiles/relay-encoder/src/lib.rs

```diff
@@ -250,23 +244,17 @@ where
 		Ok(encoded)
 	}
 
-	#[precompile::public("encodeSetController(uint256)")]
-	#[precompile::public("encode_set_controller(uint256)")]
+	#[precompile::public("encodeSetController()")]
+	#[precompile::public("encode_set_controller()")]
```

Comment:
This will break existing smart contracts, just want to make sure we handle/document this properly.

Same with `encode_nominate` and `encode_bond` above.

Reply:
Yes, this changes were made in a separate PR (https://github.com/moonbeam-foundation/moonbeam/pull/2376) that was included in this branch.